packages/taco/src/conditions/schemas/address-allowlist.ts

```diff
@@ -0,0 +1,27 @@
+import { EthAddressSchemaStrict } from '@nucypher/shared';
+import { z } from 'zod';
+
+import { baseConditionSchema, UserAddressSchema } from './common';
+
+export const AddressAllowlistConditionType = 'address-allowlist';
+
+export const addressAllowlistConditionSchema = baseConditionSchema
+  .extend({
+    conditionType: z.literal(AddressAllowlistConditionType),
+    userAddress: UserAddressSchema,
+    addresses: z
+      .array(EthAddressSchemaStrict)
+      .min(1, 'At least one address must be provided')
+      .max(25, 'A maximum of 25 addresses is allowed')
+      .describe(
+        'List of wallet addresses allowed to decrypt. Addresses should be provided in checksummed form. Matching is case-sensitive.',
+      ),
+  })
+  .strict()
+  .describe(
+    'Address Allowlist Condition for allowing decryption for specific wallet addresses. It may not be very handy on its own, but when combined with other conditions, it can be very useful.',
```

Comment:
Let's keep the docs more positive about the usefulness 😅 

```suggestion
    'Address Allowlist Condition for allowing decryption for specific wallet addresses.',
```

(This update will require a regeneration of the docs)

Reply:
Thanks for the pick. It is about pointing to the correct usage scenario with "but when combined with other conditions, it can be very useful.". But as it sounds not positive I removed now the unpossitive part 😄 . Let me know please, if you think more to be changed.